src/Traits/Categories/ButtonHelpers.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace NotificationChannels\OneSignal\Traits\Categories;
+
+use NotificationChannels\OneSignal\OneSignalButton;
+use NotificationChannels\OneSignal\OneSignalWebButton;
+
+trait ButtonHelpers
+{
+    /**
+     * Add a web button to the message.
+     *
+     * @param OneSignalWebButton $button
+     *
+     * @return $this
+     */
+    public function webButton(OneSignalWebButton $button)
+    {
+        return $this->setParameter('web_buttons', [$button->toArray()]);
+    }
+
+    /**
+     * Adds more than one web button to the message.
+     *
+     * @param array[OnSignalWebButton] $buttons
+     *
+     * @return $this
+     */
+    public function webButtons(array $buttons)
+    {
+        return $this->setParameter('web_buttons', collect($buttons)->map(function ($button) {
```

Comment:
same with this - but I've never done web buttons - so I'm sure you know what is going on here.